macros/base/start_print.cfg

```diff
@@ -174,9 +166,7 @@ gcode:
     {% endif %}
     
     {% if zcalib_plugin_enabled %}
-        {% if status_leds_enabled %}
-            STATUS_LEDS COLOR="CALIBRATING_Z"
-        {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.

Reply:
Ok done